### PR TITLE
Update AuthorNotification.md

### PR DESCRIPTION
**Pull request by v-caxian**

### CODE REVIEW

**Comment by PRMerger19** [comment]
@v-caxian : Thanks for your contribution! The author, @PRMerger16, has been notified to review your proposed change.

File: csidevdocs-public/Demo/v-caxian/AuthorNotification.md [hunk 1]
```diff
@@ -46,4 +46,4 @@ Sogup
 To effectively use the OPS Authoring extension, you must1:
 - Clone your entire repo to your local machine and keep it in sync. Functions such as link and image insertion are not reliable if the repo is out of sync.
```
Comment (PRMerger16) [inline csidevdocs-public/Demo/v-caxian/AuthorNotification.md]:
```suggestion
- Clone your entire repo to your local machine and keep it in sync. Functions such as link and image insertion are not reliable if the repo
```

**Comment by PRMerger19** [comment]
@PRMerger16 : Thanks for your contribution! The author, @PRMerger16, has been notified to review your proposed change.

File: csidevdocs-public/Demo/v-caxian/AuthorNotification.md [hunk 2]
```diff
@@ -46,4 +46,4 @@ Sogup
...
 - For accurate content preview, you need to update your VS Code settings.json file, as described under **Preview Content** below.
```
Comment (PRMerger16) [inline csidevdocs-public/Demo/v-caxian/AuthorNotification.md]:
```suggestion
- For preview, you need to update your VS Code settings.json file, as described under **Preview Content** below.
```

**Pull request closed without merging**